Healthy/Classes/Extensions/UITextField+Style.swift

```diff
@@ -0,0 +1,58 @@
+import UIKit
+
+// MARK: - TextField style
+//
+extension UITextField {
+    enum TextFieldStyle {
+        case primary
+    }
+}
+
+// MARK: - Apply textfield style
+//
+extension UITextField {
+    func applyTextFieldStyle(_ style: TextFieldStyle) {
+        NSLayoutConstraint.activate([
+            heightAnchor.constraint(equalToConstant: Constants.defaultHeight),
+        ])
+        
+        layer.cornerRadius = Constants.defaultCornerRadius
+        layer.borderWidth = Constants.defaultBorderWidth
+        layer.borderColor = Constants.defaultBorderColor.cgColor
+        
+        setLeftPaddingPoints()
+        setRightPaddingPoints()
+        
+        layer.masksToBounds = true
+    }
+}
+
+// MARK: - set right / left padding to text field
+//
+extension UITextField {
+    private func setLeftPaddingPoints(_ amount: CGFloat = Constants.defaultPadding) {
+        let paddingView = UIView(frame: CGRect(x: 0, y: 0, width: amount, height: self.frame.size.height))
+        self.leftView = paddingView
+        self.leftViewMode = .always
+    }
+    
+    private func setRightPaddingPoints(_ amount: CGFloat = Constants.defaultPadding) {
+        let paddingView = UIView(frame: CGRect(x: 0, y: 0, width: amount, height: self.frame.size.height))
+        self.rightView = paddingView
+        self.rightViewMode = .always
+    }
+}
+
+// MARK: - Constants
+//
+private extension UITextField {
+    enum Constants {
+        static let defaultCornerRadius: CGFloat = 10.0
```

Comment:
Why creating constants as an enum inside a private extension? 
If I have global constants I would create constants class with nested scopes. 
Also if you have a design system this will be redundant work to do.

Reply:
this constants just related to UITextField.
what is the best solution you mean.

Reply:
I agree with Nasr in this case, since the scope of this variable is limited to this type. Unless we have a design system, we need to keep this that way.